src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCount.java

```diff
@@ -14,6 +15,7 @@
 public final class AllelicCount implements Locatable {
     private final SimpleInterval interval;
     private final int refReadCount, altReadCount;
+    private Nucleotide refNucleotide, altNucleotide;
```

Comment:
make ```refNucleotide``` and ```altNucleotide``` private. In the constructor that does not initialize them, I guess you mean to set them to ```null```.